__tests__/cookie-banner.test.js

```diff
@@ -7,18 +7,25 @@ const PORT = configPaths.testPort
 let page
 let baseUrl = 'http://localhost:' + PORT
 
-beforeEach(async () => {
-  page = await setupPage()
-})
-
-afterEach(async () => {
-  await page.deleteCookie({ name: 'design_system_cookies_policy', url: baseUrl })
-  await page.close()
-})
-
 const COOKIE_BANNER_SELECTOR = '[data-module="govuk-cookie-banner"]'
 
 describe('Cookie banner', () => {
+  beforeEach(async () => {
+    page = await setupPage()
+  })
+
+  afterEach(async () => {
+    await page.evaluate(() => {
+      // Delete test cookies
+      var cookies = document.cookie.split(';')
+      cookies.forEach(function (cookie) {
+        var name = cookie.split('=')[0]
+        document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:00 GMT;domain=' + window.location.hostname + ';path=/'
+      })
+    })
+    await page.close()
+  })
```

Comment:
We might be able to remove the need for this code by changing the [jest-puppeteer browser context](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/README.md#jest-puppeteerconfigjs) to `incognito`

Reply:
Hm, I tried this but it didn't seem to work - lots of test failures, I assume because the cookies were persisting between tests 🤔 I don't want to spend too much more time on this, perhaps something to revisit in the future when things are a bit quieter.